src/components/tab/Tab.tsx

```diff
@@ -1,10 +1,16 @@
 import React, { FC, PropsWithChildren } from 'react';
 import { Tab as ChakraTab, TabProps } from '@chakra-ui/react';
 
-type Props = Pick<TabProps, 'isDisabled'>;
+type Props = {
+  marginX?: 'lg' | 'md';
+} & Pick<TabProps, 'isDisabled'>;
 
 const Tab: FC<PropsWithChildren<Props>> = (props) => {
-  return <ChakraTab {...props}>{props.children}</ChakraTab>;
+  return (
+    <ChakraTab marginX={props.marginX} {...props}>
+      {props.children}
+    </ChakraTab>
+  );
```

Comment:
Since you're spreading props anyway there is no need to explicitly assign `marginX` here. You could keep it as it was.